tutorials/text/text_props.py

```diff
@@ -174,13 +174,19 @@
 # |                     | this size.                                         |
 # +---------------------+----------------------------------------------------+
 #
-# The mapping between the family aliases (``{'cursive', 'fantasy',
-# 'monospace', 'sans', 'sans serif', 'sans-serif', 'serif'}``) and actual font names
+# Matplotlib can use font families installed on the user's computer, i.e.
+# Helvetica, Times, etc, as well as other user-installed fonts.
+# Font families can also be specified with generic-family aliases like
+# (``{'cursive', 'fantasy', 'monospace', 'sans', 'sans serif', 'sans-serif',
+# 'serif'}``).
+#
+# The mapping between the generic family aliases and actual font families
+# (mentioned at :ref:`default rcParams </tutorials/introductory/customizing>`)
 # is controlled by the following rcParams:
 #
 #
 # +------------------------------------------+--------------------------------+
-# | family alias                             | rcParam with mappings          |
+# | generic family alias                     | rcParam with mappings          |
```

Comment:
```suggestion
# | css family alias                        | rcParam with mappings          |
```

Maybe name this as the CSS names (to head of my question of "did Matplotlib make up these names!?").

Reply:
Maybe `CSS-style family alias`.  Many of our users do not know what css is really.

Reply:
I guess `CSS-based generic-family alias` would make more sense? (special focus on the 'generic-family')